css/style.css

```diff
@@ -15,5 +18,118 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+.navbar {
+  position: fixed;
+  right: 60px;
+  font-size: 10px;
+  width: 50%;
+}
```

Comment:
```suggestion
.navbar {
  font-size: 10px;
  width: 50%;
  font-size: 20px;
}
```

Reply:
I wonder why you wouldn't want me to fix the position? It made the bar stay on top when scrolled down. The webpage isn't responsive yet but I thought that would be a good start. Please explain why.